Source/WebCore/accessibility/AccessibilityRenderObject.h

```diff
@@ -55,8 +55,18 @@ class VisibleSelection;
 class AccessibilityRenderObject : public AccessibilityNodeObject {
 public:
     static Ref<AccessibilityRenderObject> create(AXID, RenderObject&, AXObjectCache&);
+    static Ref<AccessibilityRenderObject> create(AXID, Node&, AXObjectCache&);
     virtual ~AccessibilityRenderObject();
 
+    // Returns true if the renderer changed.
```

Comment:
Should we just name this setRendererAndReturnIfChanged?

Or maybe setRendererIfNeeded would be a more common pattern and would imply the return value?